deploy/examples/import-external-cluster.sh

```diff
@@ -76,87 +76,124 @@ function importClusterID() {
 }
 
 function importSecret() {
-  kubectl -n "$NAMESPACE" \
-    create \
-    secret \
-    generic \
-    --type="kubernetes.io/rook" \
-    "$MON_SECRET_NAME" \
-    --from-literal="$MON_SECRET_CLUSTER_NAME_KEYNAME"="$ROOK_EXTERNAL_CLUSTER_NAME" \
-    --from-literal="$MON_SECRET_FSID_KEYNAME"="$ROOK_EXTERNAL_FSID" \
-    --from-literal="$MON_SECRET_ADMIN_KEYRING_KEYNAME"="$ROOK_EXTERNAL_ADMIN_SECRET" \
-    --from-literal="$MON_SECRET_MON_KEYRING_KEYNAME"="$ROOK_EXTERNAL_MONITOR_SECRET" \
-    --from-literal="$MON_SECRET_CEPH_USERNAME_KEYNAME"="$ROOK_EXTERNAL_USERNAME" \
-    --from-literal="$MON_SECRET_CEPH_SECRET_KEYNAME"="$ROOK_EXTERNAL_USER_SECRET"
+  secret=$(kubectl -n "$NAMESPACE" get secret "$MON_SECRET_NAME")
+  if [ $? == 0 ]; then
```

Comment:
Actually I think I gave the wrong condition. If the return is 0 that should mean that kubectl found the secret, and no need to create it. Does this condition work? Please test this as I didn't test the bash syntax.
```suggestion
  if [ $? != 0 ]; then
```

Reply:
Updated

Reply:
Tested one scenario
```
runner@fv-az1228-19:~/work/rook/rook$ ./a.sh 
secret NAME   DATA   AGE
abc4   0      25s already exists
1
runner@fv-az1228-19:~/work/rook/rook$ 
runner@fv-az1228-19:~/work/rook/rook$ 
runner@fv-az1228-19:~/work/rook/rook$ export MON_SECRET_NAME=abc5
runner@fv-az1228-19:~/work/rook/rook$ 
runner@fv-az1228-19:~/work/rook/rook$ ./a.sh 
Error from server (NotFound): configmaps "abc5" not found
configmap/abc5 created
1
```